gems/pending/disk/modules/miq_disk_cache.rb

```diff
@@ -3,13 +3,13 @@
 require 'ostruct'
 
 module MiqDiskCache
-  MIN_SECTORS_PER_ENTRY = 32
-  DEF_LRU_HASH_ENTRIES  = 100
-  DEBUG_CACHE_STATS     = false
+  MIN_SECTORS_TO_CACHE = 32
```

Comment:
I was trying to be consistent with the MiqHyperVDisk module - from which this code is extracted, but you're right, the original name here is more appropriate.  I will change back.